### PR TITLE
Add ASPNET Accessibility Compat Document

### PR DESCRIPTION
**Pull request by HongGit**
Add ASP.NET .NET 4.7.1 Accessibility Compat Documentation.

### CODE REVIEW

File: Documentation/compatibility/ASPNET Accessibility Improvement.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ASP.NET is improving how ASP.NET Web Controls work with accessibility technology in Visual Studio to better support ASP.NET customers.  These includ the following changes:
+- Changes to implement missing UI accessbility patters in controls, like the Add Field dialog in Details View wizard.
+- Changes to improve display in High Contrast mode, like the Data Pager Fields Editor.
+- Improve the keyboard navigation experiences for controls, like Configure Object Context Window or Configure Data Source Window, etc.
+
+
+### Recommended Action
++__How to opt in or out of these changes__
+
++In order for the Visual Studio IDE designer to benefit from these changes, it must run on the .NET Framework 4.7.1 or later. The web application can benefit from these changes in either of the following ways:
```
Comment (rpetrusha) [inline Documentation/compatibility/ASPNET Accessibility Improvement.md]: Visual Studio IDE designer, or Visual Studio Designer?

File: Documentation/compatibility/ASPNET Accessibility Improvement.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- Install Visual Studio 2017 15.3 or later, which ships with the following AppContext Switch by default.
```
Comment (rpetrusha) [inline Documentation/compatibility/ASPNET Accessibility Improvement.md]: ships with... --> supports the new accessibility features by default. (Is this correct?)

Question: does the ASP.NET app need to target the .NET Framework 4.7.1.

File: Documentation/compatibility/ASPNET Accessibility Improvement.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+- Opt out of the legacy accessibility behaviors by adding the __Switch.UseLegacyAccessibility__ AppContext Switch to the ```<runtime>``` section in the devenv.exe.config file and setting it to false, as the following example shows.
```
Comment (rpetrusha) [inline Documentation/compatibility/ASPNET Accessibility Improvement.md]: false --> `false`

File: Documentation/compatibility/ASPNET Accessibility Improvement.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+### Source Analyzer Status
+NotPlanned
+
+
+### Change Description
+ASP.NET is improving how ASP.NET Web Controls work with accessibility technology in Visual Studio to better support ASP.NET customers.  These includ the following changes:
+- Changes to implement missing UI accessbility patters in controls, like the Add Field dialog in Details View wizard.
+- Changes to improve display in High Contrast mode, like the Data Pager Fields Editor.
+- Improve the keyboard navigation experiences for controls, like Configure Object Context Window or Configure Data Source Window, etc.
+
```
Comment (rpetrusha) [inline Documentation/compatibility/ASPNET Accessibility Improvement.md]: I think that this needs the following added:
- [X] Quirked
- [ ] Build-time break

File: Documentation/compatibility/ASPNET Accessibility Improvement.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    <configuration>
+      <runtime>
+      ...
+        <!-- AppContextSwitchOverrides value attribute is in the form of 'key1=true|false;key2=true|false  -->
+      <AppContextSwitchOverrides value="...;Switch.UseLegacyAccessibilityFeatures=false" />
+      ...
+      </runtime>
+    </configuration>
+```
+Applications that target the .NET Framework 4.7.1 or later and want to preserve the legacy accessibility behavior can opt in to the use of legacy accessibility features by explicitly setting this AppContext switch to true.
```
Comment (rpetrusha) [inline Documentation/compatibility/ASPNET Accessibility Improvement.md]: true --> 'true'

**Comment by rpetrusha** [comment]
It also occurs to me, @HongGit, that the detail on what accessibility changes were implemented is somewhat sparse. Can you add more detail?

**Comment by HongGit** [comment]
@rpetrusha I incorporated your feedback, please review the changes!

Could you please clarify a bit more on "more details"?

File: Documentation/compatibility/ASPNET Accessibility Improvement.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+.NET 4.7.1
+
+
+### Source Analyzer Status
+NotPlanned
+
+
+### Change Description
+ASP.NET is improving how ASP.NET Web Controls work with accessibility technology in Visual Studio to better support ASP.NET customers.  These include the following changes:
+- Changes to implement missing UI accessbility patterns in controls, like the Add Field dialog in the Details View wizard.
```
Comment (rpetrusha) [inline Documentation/compatibility/ASPNET Accessibility Improvement.md]: accessbility -> accessibility

**Comment by rpetrusha** [comment]
@HongGit, for more details, I was hoping that we could provide the level of detail found in PR #482, for example. If that's not possible, then we can merge this as-is.

File: Documentation/compatibility/ASPNET Accessibility Improvement.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
+
+ Quirked
+
+ Build-time breake## ASP.NET Accessibility Improvement in .NET 4.7.1
```
Comment (svick) [inline Documentation/compatibility/ASPNET Accessibility Improvement.md]: The "Quirked" and "Build-time breake" seem to be out of place here.

File: Documentation/compatibility/ASPNET Accessibility Improvement.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+- Changes to improve the keyboard navigation experiences for controls, like the Configure Object Context Window or the Configure Data Source Window.
+
+- [x] Quirked 
+- [ ] Build-time break 
+
+
+### Recommended Action
++__How to opt in or out of these changes__
+
++In order for the Visual Studio Designer to benefit from these changes, it must run on the .NET Framework 4.7.1 or later. The web application can benefit from these changes in either of the following ways:
```
Comment (svick) [inline Documentation/compatibility/ASPNET Accessibility Improvement.md]: The two lines above shouldn't start with a `+`.

Comment (HongGit) [reply Documentation/compatibility/ASPNET Accessibility Improvement.md]: Will be fixed in the next commit.

File: Documentation/compatibility/ASPNET Accessibility Improvement.md (hunk 6 above)
Comment (HongGit) [reply Documentation/compatibility/ASPNET Accessibility Improvement.md]: Will be fixed in the next commit.

File: Documentation/compatibility/ASPNET Accessibility Improvement.md (hunk 7 above)
Comment (HongGit) [reply Documentation/compatibility/ASPNET Accessibility Improvement.md]: Will be fixed in the next commit.

**Pull request merged**